fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -107,6 +107,9 @@ public class Column implements Writable {
     // Currently, analyzed define expr is only used when creating materialized views, so the define expr in RollupJob must be analyzed.
     // In other cases, such as define expr in `MaterializedIndexMeta`, it may not be analyzed after being relayed.
     private Expr defineExpr; // use to define column in materialize view
+    // For single table materialized view, record each base column for the output column.
+    @SerializedName(value = "baseColumnName")
+    private String baseColumnName;
```

Comment:
But we want to support complex expression in mv, so one mv column may reference several base columns, such as `sum(a+b)`.

Reply:
Yep. This is a compatible policy only used for sync-mv rewrite rule.  `sum(a+b)` can be created as MV but
`sum(a+b)` cannot be supported in sync mv rewrite. Maybe we can use the multi-table rewrite rule to support this.

Reply:
we will discuss it in the next pr.